### ISSUE TITLE
Main menu broken after removing Butterknife

### ISSUE THREAD

**Issue by tappdesign** [federicoiosue/Omni-Notes#805]
Tap HarmburgerIcon - Main menu is not displayed anymore, after butterknife was removed. (Only categories are displayed).   Something is wrong in MainMenuTask.java. When I reverted to older version, MainMenu is working again. It is hard to say what is wrong in current version of MainMenuTask.java.....

**Comment by federicoiosue** [comment]
Thanks for the feedback. Yes, I noticed it once I've installed the app on my device (I'm using the alpha as daily driver to check all the aspects still uncovered by the testing suite).

As you said is not that immediate to understand what's going wrong there due to binding class created by the framework itself. I have a couple of days with free time and I'm going to be back to work on that this evening.
